server/utils/nunjucksSetup.ts

```diff
@@ -213,7 +213,7 @@ export function registerNunjucks(app?: express.Express): Environment {
   })
 
   njkEnv.addFilter('dateToUnix', (date: string) => {
-    return moment(date, 'D MMM YYYY').unix()
+    return moment(date, 'YYYY-MM-DD').unix()
```

Comment:
date to unix accepts dates formatted as `YYYY-MM-DD`. This fixes the sorting of dates in nunjucks templates.